daemon/daemon_unix.go

```diff
@@ -430,6 +430,9 @@ func verifyContainerResources(resources *containertypes.Resources, sysInfo *sysi
 	if resources.BlkioWeight > 0 && (resources.BlkioWeight < 10 || resources.BlkioWeight > 1000) {
 		return warnings, fmt.Errorf("Range of blkio weight is from 10 to 1000.")
 	}
+	if resources.MaximumIOBps != 0 || resources.MaximumIOps != 0 {
+		return fmt.Errorf("Invalid qos settings: %s does not support Maximum IO Bandwidth or Maximum IO IOps", runtime.GOOS)
```

Comment:
Minor nit, I usually see it written as QoS. Lowercase seems a little odd. eg Wikipedia: https://en.wikipedia.org/wiki/Quality_of_service